src/Yaapii.Xambly/Directive/Directives.cs

```diff
@@ -610,5 +617,34 @@ public Directives Cdata(Object text)
             }
             return this;
         }
+
+        /// <summary>
+        /// Sets namespace of all current nodes selected by the cursor.
+        /// All child nodes are moved to the namespace (default).
+        /// All attributes are moved to the namespace (default).
+        /// The namespace declaration will be done in the root node.
+        /// 
+        /// If the prefix is empty a default namespace will be created.
+        /// which is declared only in the current nodes.
+        /// Attributes cannot be added to a default namespace.
+        /// 
+        /// Hint:
+        /// After declaring a namespace the XPath will be affected.
+        /// The namespace resolver is not updated by adding namespaces.
+        /// To address nodes belonging to a namespace the namesapce resolver
```

Comment:
Is this comment still correct? We do not want to inject a namespace resolver into the xambler anymore.

Reply:
No! Old version. Fixed...